infer/src/dune.in

```diff
@@ -87,7 +87,8 @@ let infer_exe_stanza =
     (if clang then "" else "-open ClangFrontendStubs")
     (if java then "" else "-open JavaFrontendStubs")
     verbose extlib_if_java
-    (if_platform_env_and_not_byte_mode "(:include linking-flags.sexp)")
+    (if windows then "(:standard -cclib -lgcc_s -cclib -lole32)"
```

Comment:
is that standard windows stuff?

Reply:
`gcc_s` comes from MinGW, `ole32` is excessively standard windows stuff.